Foundation/NSURLSession/http/EasyHandle.swift

```diff
@@ -168,6 +168,20 @@ extension _EasyHandle {
         let protocols = (CFURLSessionProtocolHTTP | CFURLSessionProtocolHTTPS)
         try! CFURLSession_easy_setopt_long(rawHandle, CFURLSessionOptionPROTOCOLS, protocols).asError()
         try! CFURLSession_easy_setopt_long(rawHandle, CFURLSessionOptionREDIR_PROTOCOLS, protocols).asError()
+        #if os(Android)
+            // See https://curl.haxx.se/docs/sslcerts.html
+            // For SSL to work you need "cacert.pem" to be accessable
+            // at the path pointed to by the URLSessionCAInfo env var.
+            // Downloadable here: https://curl.haxx.se/ca/cacert.pem
+            if let caInfo = getenv("URLSessionCAInfo")  {
```

Comment:
Without providing a URLSessionCAInfo file SSL doesn’t work at all using libcurl. It doesn’t seem to have an idea about where the android root CA certs are stored and how to use them.